instrument/gorilla.go

```diff
@@ -0,0 +1,16 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+package instrument
+
+import (
+	"github.com/gorilla/mux"
+
+	muxtrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/gorilla/mux"
+)
+
+func GorillaMuxWrapRouter(r *mux.Router) *muxtrace.Router {
```

Comment:
nit: I'd name it `WrapGorillaMuxRouter`, similar to `WrapHandler`, `WrapHandlerFunc`, `WrapHTTPClient` [here](https://github.com/DataDog/orchestrion/blob/main/instrument/http.go).